lib/ibandit/iban.rb

```diff
@@ -52,6 +59,15 @@ def bban
       iban[4..-1] unless iban.nil?
     end
 
+    def pseudo_iban
```

Comment:
Do we want to make `to_s` favour pseudo over real IBANs to allow for round-trips?